package.json

```diff
@@ -172,10 +172,10 @@
     "to-string-loader": "^1.1.5",
     "ts-node": "^3.3.0",
     "tslib": "^1.9.0",
-    "tslint": "~4.5.1",
+    "tslint": "^4.5.1",
     "tslint-loader": "^3.5.2",
     "typedoc": "^0.7.1",
-    "typescript": "~2.2.2",
+    "typescript": "^2.2.2",
```

Comment:
Is it possible to update the client's typescript to 4.0?

Reply:
@Liriaene Maybe, but that is NOT something this ticket should be responsible for. Bring that up as a requested ticket at the next team meeting if you think this would be helpful.